Formula/etcher-cli.rb

```diff
@@ -0,0 +1,31 @@
+require "language/node"
+
+class EtcherCli < Formula
+  desc "Flash OS images to SD cards & USB drives, safely and easily"
+  homepage "https://etcher.io/"
+  url "https://github.com/resin-io/etcher.git",
```

Comment:
Can you change this to use the release tarball instead ? It makes it easier to update in the long run